### ISSUE TITLE
Game list with Cover Art!!

### ISSUE THREAD

**Issue by mohsinous** [flyinghead/flycast#22]
hi @flyinghead 
can we have gamelist with cover art??
somethings like redream!!

- [X] Place an X in the box to confirm you have filled in this form

**Comment by flyinghead** [comment]
I have a work-in-progress branch:
![boxart2](https://user-images.githubusercontent.com/16261910/67633810-f33a5980-f8b4-11e9-9473-31ac6962c7bc.png)
Hopefully I'll find some time to finish it.

**Comment by mohsinous** [comment]
WOW!!! so nice

**Comment by mohsinous** [comment]
@flyinghead hi, nothing new about new MENU??

**Comment by thatman84** [comment]
Certainly would be a nice upgrade to the UI

**Comment by Gamemulatorer** [comment]
Yes the ui is ugly and also remove the sliding it's annoying replace it like ppsspp no need to hold that sliding just to go up and down in the settings :v

**Comment by flyinghead** [comment]
Thanks. I'm waiting for your code to fix all that.

**Comment by flyinghead** [comment]
Done!
